Public/Src/Engine/Scheduler/ChangeAffectedOutput/SourceChangeAffectedContents.cs

```diff
@@ -77,6 +90,14 @@ public IReadOnlyCollection<AbsolutePath> GetChangeAffectedInputs(Process process
                         changeAffectedInputs.Add(file.Path);
                     }
                 }
+
+                if (m_fileContentManager.Host.TryGetSourceSealDirectory(directory, out var sourceSealWithPatterns))

```

Comment:
I totally understand that this makes the algorithm looks more completed and would benefit other possible features in the future. 
But for QTest itself, it won't take the source code as input and instrument the source files. This part is not necessary for the change list code coverage feature.

Reply:
This feature is not for QTest only. It happens that only QTest benefits from this feature. Thus, this is for feature completeness. 

---
In reply to: [330716599](https://github.com/microsoft/BuildXL/pull/977#discussion_r330716599) [](ancestors = 330716599)